docs/dev.md

```diff
@@ -27,6 +27,7 @@ The following env variables must be set in order to deploy dev cluster on AWS:
 
 - `AWS_ACCESS_KEY_ID`
 - `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN`
```

Comment:
The `AWS_SESSION_TOKEN` variable is recommended but not required to deploy dev cluster on AWS. Let's explicitly mention that information.